.azure-pipelines/azure-pipelines-docs.yml

```diff
@@ -42,6 +42,7 @@ jobs:
       python -m pip install --no-cache-dir -r requirements-dev.txt
       python -m pip install --no-cache-dir -r requirements-optional.txt
       python -m pip install --no-cache-dir -r requirements-docs.txt
+      python -m pip install . -U
```

Comment:
I think the -U is not necessary and has no effect for local installs

Reply:
You would think that, but 

It does not matter:

    pip install -e .

It does matter (at least this is my understanding)

    pip install .

Reply:
whenever I think I finally understand pip, something like this happens :shrug:

Reply:
Nope, I was wrong. It WORKS

Reply:
is this the time I actually understand pip? :thinking:

Reply:
It might be